ihatemoney/manage.py

```diff
@@ -49,9 +49,11 @@ def run(self, config_file):
             os.path.join('conf-templates/', config_file) + '.j2'
         ).decode('utf-8')
 
+        bin_path = os.path.join(os.path.dirname(sys.executable))
```

Comment:
Why `os.path.join` ?

Reply:
Good catch. I added this while playing around, but we can get rid of it now!